README.md

```diff
@@ -138,7 +139,12 @@ The following sections provide a full list of configuration in- and output varia
 | node\_machine\_type | The instance type to use for the cluster's worker nodes | `string` | `"m5.large"` | no |
 | production\_letsencrypt | Flag to use the production environment of letsencrypt in the `jx-requirements.yml` file | `bool` | `false` | no |
 | region | The region to create the resources into | `string` | `"us-east-1"` | no |
-| spot_price | The ceiling price for spot instances | `string` | `"0.1"` | no |
+| spot\_price | The ceiling price for spot instances | `string` | `"0.1"` | no |
+| key\_name | The SSH Key Pair name | `string` | `""` | no |
+| volume\_type | The EBS Volume type | `string` | `"gp2"` | no |
+| volume\_size | The EBS Volume size in GB | `number` | `10` | no |
+| iops | The IOPS if chosen `volume_type` is `io1` | `number` | `0` | no |
+>>>>>>> feat: add an option to be able to specify ssh key pair name, ebs
```

Comment:
this seems to be some left over from a git merge?

Reply:
fixed